cypress/support/import-ui-test-data-to-firestore.js

```diff
@@ -45,100 +44,85 @@ admin
     const selectedUser = listUsersResult.users.filter(
       usr => usr.email.toLocaleLowerCase() === testUserEmail.toLocaleLowerCase()
     );
-    if (selectedUser.length === 0) {
-      const testOrg = {
-        name: testOrgName,
-        createdAt: Date()
+    if (selectedUser.length > 0) {
+      console.log("Test data are already present in the database");
+      return process.exit(0);
```

Comment:
Promise.resolve() triggers.then() and Promise.reject() triggers .error(). I could spoecify message to pass and based on that log the correct message in .then/.error sections. But I just thought adding extra conditions etc just the script even longer without real need for it. This stops the script at that very same line and even linter is happy. Hope it's fine.